Add dictionary definition for "ML" (#205)
cherryontech/website: src/assets/content/dictionary/machine-learning-ml.md

---
title: machine learning (ML)
definition: 'Machine learning is a growing technology which enables computers to learn automatically from past data. Machine learning uses various algorithms for building mathematical models and making predictions using historical data or information. Currently, it is being used for various tasks such as image recognition, speech recognition, email filtering, Facebook auto-tagging, recommender system, and many more.'
sources:
- sourceurl: https://www.javatpoint.com/machine-learning
perspectives:

> - meaning: it enables a machine to automatically learn from data, improve performance from experiences, and predict things without being explicitly programmed.

Reviewer: Looking at your changes on the site (I used the Netlify deploy preview to test), the perspective sentence isn't reading quite right. Also, we can see that the `meaning` field should not be formatting with a period at the end, since the values fill in parts of a sentence. I've included a screenshot below, but I encourage you to test your changes via either method mentioned in the [CONTRIBUTING.md](https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md).
<img width="1789" alt="image" src="https://user-images.githubusercontent.com/3057984/195480577-b8debbe6-52be-4471-b0a0-4b68f99bd5d1.png">

Can you please re-write the meaning with the full sentence structure in mind?

Author: @alliequintano done the changes and thank you for your support !!

> - meaning: As a developer , machine learnning enables a machine to automatically learn from data, improve performance from experiences, and predict things without being explicitly programmed

Reviewer: ```suggestion
- meaning: I can see my program learn from data, improve performance from experiences, and predict things without explicitly programming it to do so
```

>   role: Developer

Reviewer: ```suggestion
  role: developer
```
